x-pack/plugins/maps/public/embeddable/map_embeddable.tsx

```diff
@@ -248,6 +256,31 @@ export class MapEmbeddable
     return this._isInitialized ? this._savedMap.getAttributes().description : '';
   }
 
+  public async getFilters() {
```

Comment:
getFilters should return `[]` and getQuery should return empty query until Maps embeddable fixes [Filters applied to map visualization not preserved when added to dashboard](https://github.com/elastic/kibana/issues/91282)

The big hold out for not supporting https://github.com/elastic/kibana/issues/91282 was the lack of visibility into what the panel is displaying. With this PR, that is no longer a problem. 

Maps team will add https://github.com/elastic/kibana/issues/91282 to 8.5 roadmap and also return filters and query in getFilters and getQuery when fixing https://github.com/elastic/kibana/issues/91282

Reply:
That sounds great, thank you! I will change the return values of `getFilters` and `getQuery` temporarily as you described and add a `TODO` comment that calls out that these functions are waiting on https://github.com/elastic/kibana/issues/91282 :)

Reply:
Resolved as of 680f06d4883830426b3267caee93dd8faea9d5c2